src/scrape_up/espncricinfo/espncricinfo.py

```diff
@@ -0,0 +1,92 @@
+import requests
+from bs4 import BeautifulSoup
+
+
+class Espncricinfo:
+
```

Comment:
Add docstring in the class as well as in the methods.

You can check other modules like `GitHub`, and `IMDB` to see how the docstring is formatted.